hles/transformation/src/main/scala/org/broadinstitute/monster/dap/EnvironmentTransformations.scala

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.monster.dap
+
+import org.broadinstitute.monster.dap.environment._
+import org.broadinstitute.monster.dogaging.jadeschema.table.Environment
+
+object EnvironmentTransformations {
+
+  /** Parse all environment related fields out of a RedCap record.
+    * The schema for environment variables has been separated into 5 jade-fragments.
+    */
+  def mapEnvironment(rawRecord: RawRecord): Option[Environment] = {
+    val dog_id = rawRecord.id
+    Some(
+      Environment(
+        dogId = dog_id,
+        addressMonthYear = rawRecord.getRequired("redcap_event_name"),
+        environmentGeocoding = Some(GeocodingTransformations.mapGeocodingMetadata(rawRecord)),
```

Comment:
for my understanding, what's the purpose of wrapping these in `Some`?

Reply:
To get the Option[] type, I believe